Langs/0C0A.lang

```diff
@@ -1,59 +1,64 @@
 [MetaData]
 Language=Spanish - Spain
-Translator=97carmine
+Translator=97carmine, JeffersonRussell
 Version=2.3.0.1
+[Font]
+Extra Large=24
+Large=18
+Medium=10
+Small=8.5
 [Strings]
 Donate=Donar
-Check for Updates=Comprobar actualizaciones
-Your Windows 11 Compatibility Results are Below=Los resultados de compatibilidad de Windows 11 están abajo 
-Now Reach WhyNotWin11 via https://www.whynotwin11.org/=Ahora llega a WhyNotWin11 a través de https://www.whynotwin11.org/
-Results Based on Currently Known Requirements!=¡Resultados basados en requisitos actualmente conocidos!
+Check for Updates=Actualizar
+Your Windows 11 Compatibility Results are Below=Resultados de Compatibilidad con Windows 11
+Now Reach WhyNotWin11 via https://www.whynotwin11.org/=Obten ahora WhyNotWin11 a través de https://www.whynotwin11.org/
+Results Based on Currently Known Requirements!=¡Resultados Basados en Requisitos Actualmente Conocidos!
 Translation by=Traducción por
 Architecture (CPU + OS)=Arquitectura (CPU + SO)
-Boot Method=Método de arranque
-CPU Compatibility=Compatibilidad con CPU
-CPU Core Count=Conteo de núcleos del CPU
-CPU Frequency=Frecuencia del CPU
+Boot Method=Método de Arranque
+CPU Compatibility=Compatibilidad de CPU
+CPU Core Count=Núcleos de CPU
+CPU Frequency=Frecuencia de CPU
```

Comment:
The problem is that it does not respect grammar, it enters capital letters that should be lowercase letters

Reply:
I capitalized the same words as in the original sentence in all translations.